code/style.css

```diff
@@ -1,5 +1,360 @@
+  max-width: 200px;
+  align-self: center;
+
+}
+
+.welcome-container {
+  background-color: #2D519F;
+  padding: 50px 50px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.welcome-container .text-container {
+  background-color: white;
+  height: 100%;
+  padding: 30px;
+  margin-bottom: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+
+.shop-container {
+  background-color: #0B2D78;
+  color: white;
+  display: flex;
+  flex-direction: column;
+  padding-bottom: 30px;
+  align-items: center;
+}
+
+.shop-container .shop-cart {
+  background-color: #fff;
+  height: 100%;
+  min-width: none;
+  max-width: 400px;
+  border-radius: 20px;
+  margin: 0px 50px 50px 50px;
+  padding: 0;
+  box-shadow: 0px 12px 10px 0 #0000008f;
+  text-decoration: none;
+  display: flex;
+  flex-flow: row wrap;
+}
+
+.shop-container img {
+  width: 100%;
+  max-width: 400px;
+  border-radius: 12px 12px 0 0;
+  margin: 0;
 }
+
+.shop-container p {
+  padding: 20px;
+}
+
+.shop-container .text-container {
+  background-color: white;
+  color: black;
+  height: 100%;
+  padding: 30px;
+  margin-bottom: 30px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+
+}
+
+
+.news-container {
+  background-color: #753A90;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  color: white;
+}
+
+.news-container div {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 100%;
+  padding: 30px;
+  box-sizing: border-box;
+  margin: auto;
+}
+
+.news-container .news-text {
+  background-color: #0B2D78;
+  color: white;
+  height: 100%;
+  padding: 20px 20px;
+  margin: 50px 0px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.news-container p {
+  color: white;
+}
+
+.news-container .news-text .button {
+  background-color: white;
+  color: black;
+}
+
+.news-container img {
+  display: flex;
+  align-items: center;
+  width: 100%;
+  max-width: 300px;
+  padding: 20px;
+  border-radius: 50%;
+
+}
+
+.portfolio-container {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.gallery {
+  padding: 0px 50px;
+  display: flex;
+  justify-content: center;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.photo {
+  padding: 10px 10px 10px 10px;
+  width: 100%;
+  max-width: 300px;
+}
+
+
+.blog-container {
+  background-color: #0B2D78;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100%;
+  color: white;
+}
+
+.about-container{
+  display: flex;
+  flex-direction: column;
+  padding: 50px;
+}
+.about-container h3{
+  display: flex;
+  justify-content: center;
+}
+.tabelle{
+  display: flex;
+  flex-direction: column;
+}
+.tabelle .line{
+  display: flex;
+  flex-direction: row;
+}
+.tabelle .year{
+  padding-right: 20px;
+}
+
+
+
+/** MEDIA QUERIES **/
+
+@media (min-width: 650px) {
```

Comment:
Also great that you've work mobile first! That's best practice, and a really good thing to get used to! 😊